docs/ibexa_engage/create_campaign.md

```diff
@@ -0,0 +1,31 @@
+---
+description: Create campaign with Ibexa Engage
+edition: experience
+---
+
+# Create campaign
+
+[Campaign]([[= user_doc =]]/ibexa_engage/ibexa_engage/#campaign) is a set of concepts, divided into steps, that the user can configure.
+It includes a welcome screen, an interaction element, a form step, and an exit screen.
+Design, themes, backgrounds are all configured by Qualifio.
+
+Technically, each campaign has a unique campaign ID, that is automatically defined by the Qualifio platform when it is created.
+
+## Publication channels
+
+Every campaign includes a minimum of one publication channel that you can choose from the three options the platform provides for publishing a campaign.
+For more information about publication channels, see [Publication channel]([[= user_doc =]]/ibexa_engage/ibexa_engage/#publication-channel) in the User Documentation.
+
+## Add block in Page Builder
```

Comment:
```suggestion
## Add Campaign block in Page Builder
```
or maybe we should use "Embed/display campaign on selected landing page" - this is the goal